src/workbook/stakeholder-sheet/holdings-columns.ts

```diff
@@ -149,10 +149,32 @@ export class StockClassAsConvertedColumn {
       style: Styles.default,
     });
 
+    const conversion_value = `${
+      outstandingRange.getExtents().topLeftAddress
+    } * ${stockClass.conversion_ratio}`;
+
+    enum RoundingType {
+      CEILING = "CEILING",
+      FLOOR = "FLOOR",
+      NORMAL = "NORMAL",
+    }
```

Comment:
**Question:** What is the purpose of defining this type if it is used only in this method?

Reply:
It's not necessary actually. Just a bad habit of abstracting everything lol